### PR TITLE
Bug/c5 ingester locations

### PR DESCRIPTION
**Pull request by Dalamar42**

### CODE REVIEW

File: src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java [hunk 1]
```diff
@@ -26,28 +19,38 @@
...
+import org.slf4j.LoggerFactory;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.atlasapi.media.entity.Specialization.FILM;
 
 public class FiveBrandProcessor {
 
     private static final Logger log = LoggerFactory.getLogger(FiveBrandProcessor.class);
 
+    private static final Pattern FILM_YEAR = Pattern.compile(".*\\((\\d{4})\\)$");
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java]: comment explaining what this does

File: src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java [hunk 2]
```diff
@@ -236,11 +282,16 @@ private String childValue(Element element, String childName) {
     }
 
     private Set<String> getGenres(Element element) {
-        return genreMap.mapRecognised(ImmutableSet.of("http://www.five.tv/genres/" + element.getFirstChildElement("genre").getValue()));
+        return genreMap.mapRecognised(ImmutableSet.of(
+                "http://www.five.tv/genres/" + element.getFirstChildElement("genre").getValue())
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java]: extract magic string

File: src/main/java/org/atlasapi/remotesite/five/FiveEpisodeProcessor.java [hunk 3]
```diff
@@ -184,8 +186,9 @@ private Version getVersion(Element element) throws Exception {
         return broadcasts;
     }
 
-    private Location getLocation(Element element, String uri, Long serviceId) throws Exception {
+    private Location getLocation(Element element, String uri, Long serviceId, long playerId) {
```
Comment (AdamDM) [inline src/main/java/org/atlasapi/remotesite/five/FiveEpisodeProcessor.java]: one 'Long' and one 'long'? Keep them the same

**Comment by AdamDM** [comment]
lgtm

File: src/main/java/org/atlasapi/remotesite/five/FiveEpisodeProcessor.java (hunk 3 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/five/FiveEpisodeProcessor.java]: ack, fixing

File: src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java (hunk 1 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java]: This is preexisting. I want to minimise the changes I make for this issues

File: src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java (hunk 2 above)
Comment (Dalamar42) [inline src/main/java/org/atlasapi/remotesite/five/FiveBrandProcessor.java]: This is preexisting. I want to minimise the changes I make for this issues

**Pull request merged**